File: src/main/webapp/app/core/models/record.ts
export class Record {
  public nameClient: string;
  public police: Number;
  public birthDay: Date;
  public gender: string;

  constructor(nameClient = '', police = 0, birthDay = new Date(), gender = '') {
    this.nameClient = nameClient;
    this.police = police;
    this.birthDay = birthDay;
    this.gender = gender;
  }
}

File: src/main/java/ru/ksenia/web/rest/dto/Record.java
package ru.ksenia.web.rest.dto;

import org.joda.time.LocalDate;

public class Record {
    private String nameClient;
    public Long police;
    public LocalDate birthDay;
    public String gender;

    public String getNameClient() {
        return nameClient;
    }

    public void setNameClient(String nameClient) {
        this.nameClient = nameClient;
    }

    public Long getPolice() {
        return police;
    }

    public void setPolice(Long police) {
        this.police = police;
    }

    public LocalDate getBirthDay() {
        return birthDay;
    }

    public void setBirthDay(LocalDate birthDay) {
        this.birthDay = birthDay;
    }

    public String getGender() {
        return gender;
    }

    public void setGender(String gender) {
        this.gender = gender;
    }
}
